tests/test_lrp.py

```diff
@@ -20,8 +20,8 @@ def test_incr_counter():
     assert b"\x00\x00\x00\x00" == incr_counter(b"\xFF\xFF\xFF\xFF")
 
 
-def test_vectors_generate_plaintexts():
-    p = LRP.generate_plaintexts(b"\x56\x78\x26\xB8\xDA\x8E\x76\x84\x32\xA9\x54\x8D\xBE\x4A\xA3\xA0")
+def test_vectors_generate_plain_texts():
```

Comment:
AN12304 references this as `plaintexts`, not `plain_texts`. Consistently with the previous remarks, I would like to avoid even such subtle renaming. Could you please convert it back to `plaintexts`?

Reply:
reverted to plaintexts